packages/vm/src/runTx.ts

```diff
@@ -64,6 +64,9 @@ const journalCacheCleanUpLabel = 'Journal/cache cleanup'
 const receiptsLabel = 'Receipts'
 const entireTxLabel = 'Entire tx'
 
+// EIP-7702 flag: if contract code starts with these 3 bytes, it is a 7702-delegated EOA
+const DELEGATION_7702_FLAG = new Uint8Array([0xef, 0x01, 0x00])
```

Comment:
Shouldn't this be in `common` or else a constants section?  Feels out of place to declare a constant like this in `runTx`.